apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -0,0 +1,106 @@
+class FilesController < ApplicationController
+
+  def fs
+    request.format = 'json' if request.headers['HTTP_ACCEPT'].split(',').include?('application/json')
```

Comment:
We can defer this, but http/1.1 headers are case insensitive. I wonder if there's a more rails idiomatic way of parsing headers?